guess_the_number.py

```diff
@@ -12,13 +13,18 @@ def configure_range():
 
 def generate_secret(low, high):
     """ Generate a secret number for the user to guess """
+
     return random.randint(low, high)
 
 
 def get_guess():
     """ get user's guess, as an integer number """
-    return int(input('Guess the secret number? '))
+    while True:
```

Comment:
Code looks good, double check the spaces

Reply:
I made corrections to the invalid import and spacing.